doc/api/buffer.md

```diff
@@ -788,6 +788,28 @@ const buf = Buffer.from([0x62, 0x75, 0x66, 0x66, 0x65, 0x72]);
 
 A `TypeError` will be thrown if `array` is not an `Array`.
 
+`Buffer.from(array)` may also use the internal `Buffer` pool like
+[`Buffer.allocUnsafe()`] does.
+
+```js
+const nodeBuffers = Buffer.from([0, 1, 2, 3, 4, 5, 6, 7, 8, 9]);
+
+console.log(nodeBuffers.offset);
+// prints: 344
+
+// Default Buffer.poolsize is 8192
+console.log(Buffer.poolSize);
+// prints: 8192
+
+// But you can change it
+Buffer.poolSize = 5;
```

Comment:
This could be misleading. Changing `Buffer.poolSize` does have immediate effect on which buffers would be pooled, but it doesn't immediately change the size of the current pool, it changes the size of the __next__ pool.
It does, however, take immediate effect for determining whether should the buffer be allocated from the pool or not.